csrc/backend_ops/ncnn/pyncnn_ext/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ endif ()
 pybind11_add_module(ncnn_ext ncnn_ext.cpp)
 
 target_link_libraries(ncnn_ext PUBLIC mmdeploy_ncnn_ops ncnn)
-set_target_properties(
-        ncnn_ext PROPERTIES LIBRARY_OUTPUT_DIRECTORY
-        ${CMAKE_SOURCE_DIR}/mmdeploy/backend/ncnn)
+set(_NCNN_EXT_DIR ${CMAKE_SOURCE_DIR}/mmdeploy/backend/ncnn)
+set_target_properties(ncnn_ext PROPERTIES
+        LIBRARY_OUTPUT_DIRECTORY ${_NCNN_EXT_DIR}
+        LIBRARY_OUTPUT_DIRECTORY_DEBUG ${_NCNN_EXT_DIR}
```

Comment:
Are both `LIBRARY_OUTPUT_DIRECTORY_DEBUG` and `LIBRARY_OUTPUT_DIRECTORY_RELEASE` properties necessary?

Reply:
On MSVC, the path to the target will contain "Debug" or "Release" otherwise.